platform/admin/core/admin-core-impl/src/test/java/org/codice/ddf/admin/core/impl/ConfigurationAdminImplTest.java

```diff
@@ -222,6 +231,46 @@ public void testEnableConfigurations() throws Exception {
         "Should show the original disabled pid.",
         (String) result.get("originalFactoryPid"),
         is(TEST_FACT_PID_DISABLED));
+
+    Dictionary<String, Object> enabledProperties = new Hashtable<>();
+    enabledProperties.put(
+        org.osgi.service.cm.ConfigurationAdmin.SERVICE_FACTORYPID, TEST_FACTORY_PID);
+    enabledProperties.put(TEST_KEY_IN_METATYPE, TEST_VALUE);
+
+    verify(testConfig).delete();
+    verify(testFactoryConfig).update(enabledProperties);
+  }
+
+  /**
+   * Tests the {@link AdminConsoleService#enableConfiguration(String)} method for the case where
+   * configurationAdmin.copyConfigProperties(..) throws an Exception because no
+   * ObjectClassDefinition is found
+   *
+   * @throws Exception
+   */
+  @Test(expected = IllegalStateException.class)
+  public void testEnableConfigurationsException() throws Exception {
+    setUpListServices();
+    when(testMTI.getObjectClassDefinition(anyString(), anyString())).thenReturn(null);
```

Comment:
❓ I think the strings being passed to `getObjectClassDefinition` in this case will always be the same. Can we bound this even stricter? Or do we have to use `anyString()`?